ci-operator/step-registry/openstack/conf/createadditionalnetworks/openstack-conf-createadditionalnetworks-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml
```

Comment:
This should be SHARED_DIR. CLUSTER_PROFILE_DIR is where job-wide credentials are, we only use them in the deprecated template jobs